packages/playground/src/weblets/tf_caprover.vue

```diff
@@ -44,18 +44,15 @@
         <v-alert type="warning" variant="tonal" class="mb-6">
           <p :style="{ maxWidth: '880px' }">
             You will need to point a wildcard DNS entry for the domain you entered above to this CapRover instance IP
-            Address after deployment, otherwise, you won't be able to access the CapRover dashboard using this domain.
+            Address after deployment. Otherwise, you won't be able to access the CapRover dashboard using this domain.
           </p>
 
           <p class="font-weight-bold mt-4">
-            If you don't know what Captain root domain is, make sure to visit the
-            <a
-              target="_blank"
-              href="https://manual.grid.tf/weblets/weblets_caprover.html?highlight=caprover#caprover"
-              :style="{ color: 'inherit' }"
-            >
-              Quick start documentation.
+            If you don't know what the Captain root domain is, make sure to read the
+            <a target="_blank" href="https://manual.grid.tf/playground/caprover.html" :style="{ color: 'inherit' }">
+              quick start documentation
             </a>
+            .
```

Comment:
Add the `.` after `quick start documentation` since by adding it here it adds a space between the sentence and the `.` as seen below.

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/101194226/76679ffa-3842-471b-8ecd-7814798298eb)

```suggestion
              quick start documentation.
            </a>
```